Dockerfile.aro

```diff
@@ -1,6 +1,7 @@
 ARG REGISTRY
 FROM ${REGISTRY}/ubi8/ubi-minimal
 RUN microdnf update && microdnf clean all
+RUN curl -o /etc/pki/ca-trust/source/anchors/AMEROOT_ameroot.crt http://crl.microsoft.com/pkiinfra/certs/AMEROOT_ameroot.crt && update-ca-trust
```

Comment:
How do we know this is the correct certificate? I see it is over http and it is public. Shoud it be?

Reply:
That is a good question, I also was surprised it was not over https.  I will look into it more.

Reply:
the thumbprint is verified in Readme.md